bin/pgaudit_analyze

```diff
@@ -19,7 +19,8 @@ use POSIX qw(setsid);
 use lib dirname($0) . '/../lib';
 use PgAudit::CSV;
 use PgAudit::Wait;
-
+# flush the output buffer
+$|++;
```

Comment:
Well, this is quite the Perl-ism. What is the purpose of this exactly? Any way we can make this clearer / more English?

Reply:
I added it b/c when testing the code as a systemd service the prints were not making it to the logs while I was watching.

$|++ sets $| to 1, default is 0

perhaps:
# flush the output buffer so print statements occur immediately in the log file

From https://www.perlmonks.org/?node=perlman%3Aperlvar

_$|
If set to nonzero, forces a flush right away and after every write or print on the currently selected output channel. Default is 0 (regardless of whether the channel is actually buffered by the system or not; $| tells you only whether you've asked Perl explicitly to flush after each write). Note that STDOUT will typically be line buffered if output is to the terminal and block buffered otherwise. Setting this variable is useful primarily when you are outputting to a pipe, such as when you are running a Perl script under rsh and want to see the output as it's happening. This has no effect on input buffering. (Mnemonic: when you want your pipes to be piping hot.)_

Reply:
OK, but maybe we can use `$OUTPUT_AUTOFLUSH` instead. That may require `use English` but I'm fine with that.

The main thing is if you try to search for `$|++` you'll get nothing. At least that's what happened when I tried.

Reply:
But searching for `$|` does work so that was my bad. Even so...

Reply:
`$|++` is a really common Perl-ism
I've never seen `$OUTPUT_AUTOFLUSH` in use in the wild, but probably a case of observation bias.
My preference is to use `$|++` with a good comment to reduce dependencies but fine either way.

Reply:
OK, just add a mention of `$OUTPUT_AUTOFLUSH` in the comment to make this easier to find.

In other projects I've dealt with this issue by just using syswrite(), so that's another option.

Reply:
BTW - I am waiting for a reply here or a change before continuing my review. Just in case that was not clear.

Reply:
added reference to `$OUTPUT_AUTOFLUSH` in comment